src/translations/en.json

```diff
@@ -954,5 +952,16 @@
   "UserCard.showFullBioLink": "more",
   "UserCard.viewProfileLink": "View profile",
   "UserDisplayName.banned": "Banned user",
-  "UserDisplayName.deleted": "Deleted user"
-}
+  "UserDisplayName.deleted": "Deleted user",
+  "EditListingWizard.tabLabelCapacity": "Capacity",
+  "EditListingCapacityPanel.createListingTitle": "Edit The Capacity of Comfy sauna",
+  "EditListingCapacityForm.capacityPlaceholder": "Chosen your capacity",
+  "EditListingCapacityForm.capacityRequired": "Required Chonsen",
```

Comment:
:)) Kiểm tra lại tiếng Anh đoạn này

Reply:
Em sửa lại:
"EditListingWizard.tabLabelCapacity": "Capacity",
  "EditListingCapacityPanel.createListingTitle": "Edit The Capacity of Sauna",
  "EditListingWizard.saveEditCapacity": "Save change",
  "EditListingCapacityForm.capacityPlaceholder": "Choose your capacity",
  "EditListingCapacityForm.capacityRequired": "Required Choose",

Reply:
Thanks anh ^^!